thirdparty/jaspr_bootstrap/lib/src/components/carousel.dart

```diff
@@ -62,7 +62,8 @@ class Carousel extends BaseComponent {
         if (attributes != null) ...attributes!,
       },
       children: [
-        if (enableIndicator) _CarouselIndicator(componentId: componentId, items: items),
+        if (enableIndicator)
+          _CarouselIndicator(componentId: componentId, items: items),
```

Comment:
Why are you editing Carousel component in one this commit where should be edited only Table component?

Reply:
Oops sorry, auto formatting.

Reply:
It is a good practice to check changed code before make the commit. :)

Probably we should setup same formatting in our editors for now.. 
I am using Android Studio Arctic Fox with default formatting setup:
![image](https://user-images.githubusercontent.com/2686906/177829944-f0116e83-70d3-490a-9ec2-4d34b2ab4c82.png)

So I hope it can help in better setup on your side.. :)